google-api-client/OVERVIEW.md

```diff
@@ -1,36 +1,45 @@
-# Google API Client Library for Ruby Docs
+# Monolithic Client for Google APIs (DEPRECATED)
 
-The Google API Client Library for Ruby is designed for Ruby
-client-application developers. It offers simple, flexible
-access to many Google APIs.
+This library includes REST clients for all Discovery-based Google APIs as of December 2020.
 
-## Features
+This library is **deprecated** and will likely not be updated in the future. Instead, please install the gem corresponding to the specific service you want to use. For example, to use the Google Drive V3 client, install the `google-apis-drive_v3` gem.
 
```

Comment:
What do you think about adding a sentence here mentioning the extracted gems? (`google-apis-core` and `google-apis-generator`?)

Reply:
Added a sentence regarding generator. Core should just be a dependency (like google-cloud-core), and I don't expect people would install it directly.